src/Schema/Conveyor/MetadataSchemaConveyor.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Cycle\Schema\Conveyor;
+
+use Cycle\Annotated\Embeddings;
+use Cycle\Annotated\Entities;
+use Cycle\Annotated\MergeColumns;
+use Cycle\Annotated\MergeIndexes;
+use JetBrains\PhpStorm\ExpectedValues;
+use Spiral\Attributes\ReaderInterface;
+use Spiral\Tokenizer\ClassLocator;
+use Symfony\Component\Finder\Finder;
+use Yiisoft\Aliases\Aliases;
+use Yiisoft\Yii\Cycle\Exception\EmptyEntityPathsException;
+
+abstract class MetadataSchemaConveyor extends SchemaConveyor
```

Comment:
Why class `MetadataSchemaConveyor` is separately?
We can merge code from  `MetadataSchemaConveyor` to `SchemaConveyor` and make  `SchemaConveyor` abstract.

Reply:
Because the list of generators can be different and can be related not only to the metadata parsing

Reply:
renamed